data/schema/schema_template.yaml

```diff
@@ -25,9 +25,9 @@ region:
   source: GeoRegion.source # OPTIONAL - see GeoRegion docs in types.yaml
```

Comment:
something I just noted: just a line above this, would you mind to change from 
```
source_url: GeoRegion.source_url # OPTIONAL - see GeoRegion in types.yaml
```
to
```
source_url: GeoRegion.source_url # OPTIONAL - see GeoRegion docs in types.yaml
```

(added " docs") Thanks!